examples/notebooks/integrate_3rd_transforms.ipynb

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
the file name could be `integrate_3rd_party_transforms.ipynb`

Reply:
updated in the latest commit.
Thanks.